site/sfguides/src/data_collaboration_native_app/data_collaboration_native_app.md

```diff
@@ -0,0 +1,630 @@
+```
+
+Next we create two [Virtual Warehouses](https://docs.snowflake.com/en/user-guide/warehouses-overview) that can be used to load the initial dataset, and do our model training respectively.
+
+```SQL
+-- Create a virtual warehouse for loading and ad-hoc queries
+CREATE OR REPLACE WAREHOUSE QUERY_WH WITH 
+  WAREHOUSE_SIZE = 'X-SMALL' 
+  WAREHOUSE_TYPE = 'STANDARD' 
+  AUTO_SUSPEND = 300 
+  AUTO_RESUME = TRUE 
+  MIN_CLUSTER_COUNT = 1 
+  MAX_CLUSTER_COUNT = 1;
+
+-- Create a virtual warehouse for training
+CREATE OR REPLACE WAREHOUSE training_wh with 
+  WAREHOUSE_SIZE = 'MEDIUM' 
+  WAREHOUSE_TYPE = 'snowpark-optimized' 
+  MAX_CLUSTER_COUNT = 1
+  AUTO_SUSPEND = 300 
+  AUTO_RESUME = TRUE;
+```
+
+### Load Data 
+
+Next we will create a database and schema that will house the tables to train our model.
+
+```SQL
+-- Create the application database and schema
+CREATE OR REPLACE DATABASE NATIVE_APP_DEMO;
+CREATE OR REPLACE SCHEMA NATIVE_APP_DEMO;
+USE SCHEMA NATIVE_APP_DEMO.NATIVE_APP_DEMO;
+```
+
+Our data is in Parquet format, so we will create a file format object
+
+```SQL
+CREATE OR REPLACE FILE FORMAT parquet_format
+TYPE = PARQUET;
+```
+
+Next we set up our [External Stage](https://docs.snowflake.com/en/user-guide/data-load-s3-create-stage#external-stages) to our data. In our business scenario, we would have a secure [Storage Integration](https://docs.snowflake.com/en/sql-reference/sql/create-storage-integration) to the external stage rather than a public s3 bucket.
+
+```SQL
+CREATE OR REPLACE STAGE quickstart_cc_default_training_data
+    URL = 's3://sfquickstarts/two_way_data_share/train/'
+    FILE_FORMAT = parquet_format;
+```
+
+This DDL will create the structure for the table which is the main source of data for our lab.
+
+```SQL
+CREATE OR REPLACE TABLE cc_default_training_data
+  USING TEMPLATE (
+    SELECT ARRAY_AGG(OBJECT_CONSTRUCT(*))
+      FROM TABLE(
+        INFER_SCHEMA(
+          LOCATION=>'@quickstart_cc_default_training_data',
+          FILE_FORMAT=>'parquet_format'
+        )
+      ));
+```
+
+Now we will load the data in the tables. We can scale up the warehouse temporarily so we do not have to wait as long
+
+```SQL
+ALTER WAREHOUSE query_wh SET warehouse_size=MEDIUM;
+```
+
+The code below loads the data in to the tables.
+
+```SQL
+COPY INTO cc_default_training_data 
+  FROM @quickstart_cc_default_training_data 
+  FILE_FORMAT = (FORMAT_NAME= 'parquet_format') 
+  MATCH_BY_COLUMN_NAME=CASE_INSENSITIVE;
+```
+We can now scale down the warehouse since we have finished loading data for the lab
+
+```SQL
+ALTER WAREHOUSE query_wh SET warehouse_size=XSMALL;
+```
+
+You should have loaded over 5.5 million rows in a few minutes. To check, query the data in the worksheet
+
+```SQL
+SELECT COUNT(*) 
+  FROM CC_DEFAULT_TRAINING_DATA;
+```
+
+We have loaded all the data in Zamboni. We can now proceed with training the model
+
+<!-- ------------------------ -->
+## Provider Account (Zamboni) - Create Model
```

Comment:
@sfc-gh-tbuchhorn Rename this to "Create Model" and add "Provider Account (Zamboni)" sub-header underneath.